tests/MongoDB.Driver.Core.Tests/Core/Operations/ChangeStreamCursorTests.cs

```diff
@@ -37,6 +37,10 @@ namespace MongoDB.Driver.Core.Operations
 {
     public class ChangeStreamCursorTests : OperationTestBase
     {
+        #region static
+        private static readonly SemanticVersion __serverVersion = new SemanticVersion(4, 0, 0);
```

Comment:
Why `4.0.0`?

Can you either name the variable to indicate why or add a comment to indicate why?

Reply:
it's just a random value to use as arguments for constructors. I've changed it a bit